### ISSUE TITLE
Heroku deployment fails

### ISSUE THREAD

**Issue by norbertogomez** [symfony/symfony-demo#642]
Hello, while trying to deploy the project to heroku using the button that provided in description I got this:

```
PHP Fatal error:  Uncaught Error: Class 'Symfony\Component\Dotenv\Dotenv' not found in /app/bin/console:20
Stack trace:
#0 {main}
  thrown in /app/bin/console on line 20
```

Any Ideas?

**Comment by stof** [comment]
you need to set the `APP_ENV` environment variable when deploying on heroku

**Comment by stof** [comment]
See #643

**Comment by javiereguiluz** [comment]
Closing as fixed by #643.
